bigquery_etl/stripe/event.schema.json

```diff
@@ -5068,7 +5068,7 @@
                 "type": "TIMESTAMP"
               },
               {
-                "name": "marked_uncollectable_at",
+                "name": "marked_uncollectible_at",
```

Comment:
Does this field already exist? I expect we need to maintain the old name unless you're planning to rewrite the table to drop the old column.

Reply:
i rewrote the tables to drop the incorrectly named column, there was no data in the field until today